Catalog changelog 03 07 2024 (#3390)
SkylineCommunications/dataminer-docs: release-notes/Cloud/DCP_change_log.md

> [!NOTE]
> Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).

> #### 3 July 2024 - New Feature - Catalog - Version 2 of the User Catalog API

Author: The API should be exposed to the public so would probably be beneficial to have it on there aswell.

> #### 3 July 2024 - New Feature - Catalog - Version 2 of the User Catalog API

Reviewer: OK, I'll look into adding the API in the user guide. Another question: does this RN have an intranet ID, or is there no intranet entry for this one?

  Verify if a catalog item version can be deployed on the available dataminer systems
- **POST api/user-catalog/v2-0/catalogs/{catalogId}/ranges/versions/{versionId}/deploy** (UPDATED)

  Deploy a specific version of a specific catalog item.
- **POST api/user-catalog/v2-0/catalogs/register**

  Register a catalog item.
- **PATCH api/user-catalog/v2-0/catalogs/{catalogId}/publishing-state?organizationId={organizationId}**

  Set the publishing state of a catalog item
- **PATCH api/user-catalog/v2-0/catalogs/{catalogId}/version/{versionId}/publishing-state?organizationId={organizationId}**

  Set the publishing state of a catalog version
- **PATCH api/user-catalog/v2-0/catalogs/{catalogId}/ranges/{rangeId}/state?organizationId={organizationId}**

  Set the state of a specific range of a specific catalog item.
- **PATCH api/user-catalog/v2-0/catalogs/{catalogId}/versions/{versionId}/state?organizationId={organizationId}**

  Set the state of a specific version of a specific catalog item.

#### 3 July 2024 - New Feature - Catalog - Change state support [ID_40096]

> When viewing an item on the catalog, it will now be possible to change the state of a range or version.

Author: Currently only Skyline members are allowed to change the state.

> When viewing an item on the catalog, it will now be possible to change the state of a range or version.

Reviewer: ```suggestion
Skyline employees will now be able to change the state of a range or version when they view an item in the Catalog.
```

#### 3 July 2024 - New Feature - Catalog - Custom tag support [ID_40030]

> When viewing an item on the catalog, it will now be possible to give a custom tag to ranges.

Author: Only users that have access to that organization will be able to update/delete custom tags on ranges

> When viewing an item on the catalog, it will now be possible to give a custom tag to ranges.

Reviewer: What do you mean with "that organization"? Are the tags linked to organizations?

> When viewing an item on the catalog, it will now be possible to give a custom tag to ranges.

Author: Everyone is able to view the tags, but only the users that are part of the organization that published the item can update/delete tags

#### 3 July 2024 - New Feature - Catalog - Version history [ID_39903]

When viewing an item on the catalog, the versions tab will now include a version history section.
The version history section will by default show all supported ranges and versions.

> A toggle has also been introduced to include unsupported ranges and versions.

Author: Indeed the list is what you can expect toggling the toggle button
